src/components/UsefulInfo/styles.scss

```diff
@@ -0,0 +1,7 @@
+
+.list-items-grid {
+  display: grid;
+  grid-row: auto;
+  grid-gap: 30px;
+  padding-right: 30px;
+}
```

Comment:
I don't think we need this because SearchResultListItem is already a div with display block. Maybe a margin top for every SearchResultListItem is enough.